contracts/staking/src/execute.rs

```diff
@@ -711,6 +719,23 @@ pub fn recover(
         .add_submessage(sub_msg))
 }
 
+pub fn update_config_from_migrate(
+    deps: DepsMut,
+    msg: MigrateMsg,
+) -> ContractResult<Response> {
+    let mut config: Config = CONFIG.load(deps.storage)?;
+   
+    // update oracle contract address v2
+    if msg.oracle_contract_address_v2.is_some() {
+        let oracle_contract_address_v2 = msg.oracle_contract_address_v2.unwrap();
+        validate_address(&oracle_contract_address_v2, "osmo")?;
+        config.oracle_contract_address_v2 = Some(Addr::unchecked(oracle_contract_address_v2));
```

Comment:
Considering the `?` operator returns `x` when applied to `Ok<x>`, or immediately returns the `Err` to the calling function, I guess we can re-write this code to avoid having to use the `unchecked` method: 

```suggestion
        let address = validate_address(&oracle_contract_address_v2, "osmo")?;
        config.oracle_contract_address_v2 = Some(address);
```


References: 
* [The question mark operator](https://doc.rust-lang.org/reference/expressions/operator-expr.html#the-question-mark-operator)
* [`StdResult` definition](https://docs.rs/cosmwasm-std/latest/cosmwasm_std/type.StdResult.html)

As you can see the `?` operator can be applied to `Result<T, E>` type, and `StdResult<T>` is defined as `Result<T, StdErr>` so this should work properly.